_about/CONTRIBUTING.md

```diff
@@ -42,21 +42,21 @@ Be sure to run `pre-commit install` before starting any development. `pre-commit
 
 ### Building the provider
 
-Anytime you want to test a local change, run the build command, which creates the provider binary in the `./build` folder
+Anytime you want to test a local change, run the build command, which creates the provider binary in the `.build/` folder:
```

Comment:
See https://github.com/PrefectHQ/terraform-provider-prefect/pull/555/commits/880a7da3f38d395594204ee54fea39d579a2c716 for an explanation why this moved